tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py

```diff
@@ -95,25 +119,236 @@ def _is_whitespace(c):
+                )
+
+                if self.tokenizer.pad_token_id in encoded_dict["input_ids"]:
+                    non_padded_ids = encoded_dict["input_ids"][: encoded_dict["input_ids"].index(self.tokenizer.pad_token_id)]
+                else:
+                    non_padded_ids = encoded_dict["input_ids"]
+
+                tokens = self.tokenizer.convert_ids_to_tokens(non_padded_ids)
+
+                token_to_orig_map = {}
+                for i in range(paragraph_len):
+                    index = len(truncated_query) + sequence_added_tokens + i if self.tokenizer.padding_side == "right" else i
+                    token_to_orig_map[index] = tok_to_orig_index[len(spans) * self.doc_stride + i]
+
+                encoded_dict["paragraph_len"] = paragraph_len
+                encoded_dict["tokens"] = tokens
+                encoded_dict["token_to_orig_map"] = token_to_orig_map
+                encoded_dict["truncated_query_with_special_tokens_length"] = len(truncated_query) + sequence_added_tokens
+                encoded_dict["token_is_max_context"] = {}
+                encoded_dict["start"] = len(spans) * self.doc_stride
+                encoded_dict["length"] = paragraph_len
+
+                spans.append(encoded_dict)
+
+                if "overflowing_tokens" not in encoded_dict:
+                    break
+                span_doc_tokens = encoded_dict["overflowing_tokens"]
+
+            for doc_span_index in range(len(spans)):
+                for j in range(spans[doc_span_index]["paragraph_len"]):
+                    is_max_context = _new_check_is_max_context(spans, doc_span_index, doc_span_index * self.doc_stride + j)
+                    index = (
+                        j
+                        if self.tokenizer.padding_side == "left"
+                        else spans[doc_span_index]["truncated_query_with_special_tokens_length"] + j
+                    )
+                    spans[doc_span_index]["token_is_max_context"][index] = is_max_context
+
+            for span in spans:
+                # Identify the position of the CLS token
+                cls_index = span["input_ids"].index(self.tokenizer.cls_token_id)
+
+                # p_mask: mask with 1 for token than cannot be in the answer (0 for token which can be in an answer)
+                # Original TF implem also keep the classification token (set to 0) (not sure why...)
+                p_mask = np.array(span["token_type_ids"])
+
+                p_mask = np.minimum(p_mask, 1)
+
+                if self.tokenizer.padding_side == "right":
+                    # Limit positive values to one
+                    p_mask = 1 - p_mask
+
+                p_mask[np.where(np.array(span["input_ids"]) == self.tokenizer.sep_token_id)[0]] = 1
+
+                # Set the CLS index to '0'
+                p_mask[cls_index] = 0
+
+                span_is_impossible = example['is_impossible']
+                start_position = 0
+                end_position = 0
```

Comment:
I do not understand these 2 lines, why do you need put zeros to variables and then to representation?

Reply:
These variables can be removed